c10/util/Exception.h

```diff
@@ -740,14 +733,4 @@ inline void deprecated_AT_ASSERTM() {}
     C10_EXPAND_MSVC_WORKAROUND(TORCH_INTERNAL_ASSERT(cond, __VA_ARGS__)); \
   } while (false)
 
-// Deprecated alias; this alias was deprecated because it represents extra API
-// surface that makes it hard for people to understand what macro to use.
-// Use TORCH_CHECK(false, ...) or TORCH_INTERNAL_ASSERT(false, ...) to
-// unconditionally fail at a line of code.
-#define AT_ERROR(...)                                                        \
```

Comment:
I'm afraid there are too many users of this outside of PyTorch core to remove it right now: https://github.com/search?q=%2F%28%3F-i%29AT_ERROR%5C%28%2F&type=code

Please add this API back for now, I don't think the churn for users is worth the maintenance burden of this function at the moment.

Reply:
I agree with you totally.
There are so many usages of the three API out of PyTorch core, so it`s not the right time to remove those, I will close those three PR now, thank you for your review.

Reply:
Sure, I will update the pr right now :D